### PR TITLE
Add data name to message

### PR DESCRIPTION
**Pull request by uekerman**
@BenjaminRueth Could you please give it a try?

### CODE REVIEW

File: src/cplscheme/config/CouplingSchemeConfiguration.cpp [hunk 1]
```diff
@@ -1052,8 +1051,19 @@ void CouplingSchemeConfiguration::checkIfDataIsExchanged(
...
+
+  std::string dataName = "";
+  for (mesh::PtrMesh mesh : _meshConfig->meshes()) {
+    for (mesh::PtrData data : mesh->data()) {
+      if (data->getID() == dataID) {
+        dataName = data->getName();
+        break;
+      }
+    }
+  }
```
Comment (fsimonis) [inline src/cplscheme/config/CouplingSchemeConfiguration.cpp:1063]: Can you refactor this into a little helper function?
Maybe `meah::PtrData CouplingSchemeConfiguration::findDataByID(int ID);`
Return `nullptr` if there is no match. Then this becomes:

```
auto dataptr = findDataByName(dataName);
if (dataptr) {
  std::string dataName = dataptr->getName();
}
```

Comment (uekerman) [reply src/cplscheme/config/CouplingSchemeConfiguration.cpp:1063]: But so far we only need this here. Isn't that a bit over-engineered?

Comment (fsimonis) [reply src/cplscheme/config/CouplingSchemeConfiguration.cpp:1063]: It's all about mental overhead at the call side. _(the inner loop should probably be a `findDataByID`in `Mesh`)_

Comment (BenjaminRueth) [reply src/cplscheme/config/CouplingSchemeConfiguration.cpp]: Done. See https://github.com/precice/precice/pull/826/commits/5413f41808606c87dc4e192ab30e63240e82284c.

**Comment by BenjaminRueth** [comment]
Can be merged as soon as Travis is done.

**Pull request merged**